datahub-web-react/src/app/ingest/source/builder/RecipeForm/snowflake.ts

```diff
@@ -0,0 +1,136 @@
+import { FieldType, RecipeField, setListValuesOnRecipe } from './common';
+
+export const SNOWFLAKE_ACCOUNT_ID: RecipeField = {
+    name: 'account_id',
+    label: 'Account ID',
+    tooltip: 'Snowflake account. e.g. abc48144',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.account_id',
+    rules: null,
+};
+
+export const SNOWFLAKE_WAREHOUSE: RecipeField = {
+    name: 'warehouse',
+    label: 'Warehouse',
+    tooltip: 'Snowflake warehouse.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.warehouse',
+    rules: null,
+};
+
+export const SNOWFLAKE_USERNAME: RecipeField = {
+    name: 'username',
+    label: 'Username',
+    tooltip: 'Snowflake username.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.username',
+    rules: null,
+};
+
+export const SNOWFLAKE_PASSWORD: RecipeField = {
+    name: 'password',
+    label: 'Password',
+    tooltip: 'Snowflake password.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.password',
+    rules: null,
+};
+
+export const SNOWFLAKE_ROLE: RecipeField = {
+    name: 'role',
+    label: 'Role',
+    tooltip: 'Snowflake role.',
+    type: FieldType.TEXT,
+    fieldPath: 'source.config.role',
+    rules: null,
+};
+
+const schemaAllowFieldPath = 'source.config.schema_pattern.allow';
+export const SNOWFLAKE_SCHEMA_ALLOW: RecipeField = {
+    name: 'schema_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip: 'Use regex here.',
+    placeholder: '^my_db\\.my_schema$',
```

Comment:
same comment about just one `\` and same goes to the fields below here

Reply:
In UI, only one \ will be visible.